airflow/utils/dates.py

```diff
@@ -98,7 +109,7 @@ def date_range(
     return sorted(l)
 
 
-def round_time(dt, delta, start_date=datetime.min):
+def round_time(dt, delta, start_date=timezone.make_aware(datetime.min)):
```

Comment:
looks like this is the root cause of the `default_timezone=system` bug, `timezone.make_aware(datetime.min)` throws that exception.

Reply:
I think the issue is somewhere else (init of timezone with system), but let me have a look.

Reply:
Fixed.